docs/development/debugging.md

```diff
@@ -39,19 +39,31 @@ This will allow you to observe call chains and identify potential issue areas.
 
 > Note that this will increase the size of the build significantly, taking up around 50G of disk space
 
-Write the following file to `electron/.git/info/exclude/debug.gn`
+Write the following file to `electron/build/args/debug.gn`
```

Comment:
I don't think this is what we want to suggest, we don't want folks adding files into the repo that aren't gitignored by the main repo.  It'll result in some very strange linting / formatting behavior.

Reply:
If that is the case, do you suggest that maybe we just use the command line here to generate debugging build here? As I said, the instruction is not correct, the `.git/info/exclude` is a regular file instead of a directory.

a command to generate debug build without gn file:`gn gen out/Debug --args="import(\"//electron/build/args/testing.gn\") is_debug=true symbol_level=2 forbid_non_component_debug_builds=false"`